### DISCUSSION TITLE
Manage general users?

### DISCUSSION THREAD

**Discussion by Gentatsu** [Glench/ExtPay#152] [General]
Not sure if it's because I'm just testing it, but I don't see a way to manage general users (and potentially delete them). I'd like to test the `onPaid` functionality by creating a user in my backend which I then issue a JWT to do further API requests with. There'll likely be a bit of debugging required for this, so it would be handy to be able to delete myself and reregister to test/emulate this.

**Comment by Glench** [comment]
It sounds like you're wanting to do something that ExtPay isn't designed for. You can certainly look at using our backend API but it is not officially supported and may change without any announcement.
